Tests/WorkspaceTests/PinsStoreTests.swift

```diff
@@ -31,7 +31,24 @@ final class PinsStoreTests: XCTestCase {
         let pin = PinsStore.Pin(package: foo, repository: fooRepo, state: state, reason: "bad")
         // We should be able to round trip from JSON.
         XCTAssertEqual(try PinsStore.Pin(json: pin.toJSON()), pin)
-        
+
+        var lsf = Basic.localFileSystem
+        let tmpDir = try TemporaryDirectory(prefix: #function, removeTreeOnDeinit: true)
```

Comment:
We have a method utility function `mktmpdir` which does the same thing but is easier to use.